assets/app/view/game/part/city.rb

```diff
@@ -127,6 +112,86 @@ class City < Base
           pointy: [nil, 42, 62, 57],
         }.freeze
 
+        ANGLE_UPPER_RIGHT = -60
+        ANGLE_LOWER_RIGHT = 10
+        ANGLE_LOWER_LEFT = 170
+        ANGLE_UPPER_LEFT = -120
+
+        REVENUE_LOCATIONS_BY_EDGE = {
+          0 => [
+                { regions: [19], angle: ANGLE_LOWER_LEFT },
```

Comment:
your indenting is not great here, do you have an auto indenter?

Reply:
No. I'm embarrassed to admit I just use vim. I'll see if there is better Ruby support I can add.

Reply:
vim is all i use